playground/CommandInfoUpdater/GarnetCommandsDocs.json

```diff
@@ -0,0 +1,360 @@
+[
```

Comment:
If this is not a built-in resource (in order to save space), should we not include it in the NuGet somehow?

Reply:
I'm assuming you're referring to RespCommandsDocs.json (in Garnet.server/Resp), which is an embedded resource (similiar to RespCommandsInfo.json), that get lazily parsed upon request. Is that not what we wanted here?